pagefaultgames/pokerogue
# [Feature] Implement evil teams

## What are the changes?
Enables evil team grunts and leader to spawn

## Why am I doing these changes?
* Evil teams are a commonly requested feature
* Classic mode is lacking content in the midgame apart from a few rival battles
* Players only need to prepare for the final boss and the rival as the main hurdles in a classic playthrough. This introduces some roguelike randomness where there is another major boss that players have to prepare for

## What did change?
* Uncommented lines in classicFixedBattles
* Changed the `getEvilGruntPartyTemplate` function so it would no longer crash since it checks for the waveIndex of both 61 and 62 for an encounter on floor 62. Switched to just using if/else instead.

### Screenshots/Videos
![](https://cdn.discordapp.com/attachments/1249574581279391756/1249584429760905236/image.png?ex=6667d5b0&is=66668430&hm=ff853c5a9d102307c02be488d4932ff0f648a6d42b1b3c805a62c207b9820476&)

## How to test the changes?
Start a classic run and check out the new encounters on floors 35, 62, 64, 66 ,112, 114, 115, and 165
You can also use overrides to jump to those floors

## Checklist
